proposals/stage-1/PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -18,100 +18,140 @@ Projects must be proposed via GitHub. Project proposals submitted to the _$FOUND
+ * Explicitly define a project governance and committer process. This is preferably laid out in a GOVERNANCE.md file and references a CONTRIBUTING.md and OWNERS.md file showing the current and emeritus committers.
  * Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website).
- * Receive a supermajority vote from the CPC to move to Graduation stage. Projects can attempt to move directly from Sandbox to Top Level, if they can demonstrate sufficient maturity. 
+ * Other metrics as defined by the applying Project during the application process in cooperation with the CPC.
+ * Receive a supermajority vote from the CPC to move to Impact stage. Projects can move directly from At Large to Impact, if they can demonstrate sufficient maturity and have met all requirements. 
 
 
-### Archive/Emeritus Stage*
+### Emeritus Stage
 **Definition**
 
+Emeritus projects are projects which the maintainers feel have reached or are nearing end-of-life. Emeritus projects have contributed to the ecosystem, but are not necessarily recommended for modern development as there may be more actively maintained choices. The _$FOUNDATION_ appreciates the contributions of these projects and their communities, and the role they have played in moving the ecosystem forward. 
+
 **Examples**
 
+1. Projects that are "complete" by the maintainers' standards.
+1. Projects that do not plan to release major versions in the future.
+
 **Expectations**
-Projects in this stage are not in active development. The foundation will continue to hold the IP and any trademarks and domains, but the project does not draw on foundation resources. 
+
+Projects in this stage are not in active development. Their maintainers may infrequently monitor their repositories, and may only push updates to address security issues, if at all. Emeritus projects should clearly state their status and what any user or contributor should expect in terms of response or support. If there is an alternative project the maintainers recommend, it should be listed as well. The foundation will continue to hold the IP and any trademarks and domains, but the project does not draw on foundation resources. 
 
 **Acceptance Criteria**
-Archiving criteria: projects may be archived via a 2/3 vote from the CPC and with approval from project ownership." In cases where there is a lack of project ownership, only a 2/3 vote from the CPC is required
+
+Projects may be granted Emeritus status via a 2/3 vote from the CPC and with approval from project ownership. In cases where there is a lack of project ownership, only a 2/3 vote from the CPC is required.
 
 
 ## IV. Annual Review Process
 
-_TBD_
+The CPC shall develop an annual review process to determine whether projects are in the stage that accurately reflects their needs and goals. 
```

Comment:
I think this might address my above question. sorry on mobile

Reply:
TY @boneskull  <3